client/ayon_core/hosts/houdini/api/plugin.py

```diff
@@ -147,7 +147,6 @@ def cache_instance_data(shared_data):
     def create_instance_node(
         folder_path, node_name, parent, node_type="geometry"
     ):
-        # type: (str, str, str) -> hou.Node
```

Comment:
This is valid type-hinting.

Reply:
Because the docstring is **also** defining type hints but with more details - so they would essentially be redundant.
Plus this type-hint was actually invalid, because it missed one input argument.